tests/wasm.rs

```diff
@@ -255,3 +259,29 @@ async fn get_put() {
 
     assert_eq!(dispatch(db, "close", "").await.unwrap(), "");
 }
+
+// We can't run a web server in wasm-in-the-browser so this is the next
+// best thing: a manual test that FETCHES OVER THE NETWORK. To run it:
+// 1. uncomment it out
+// 2. wasm-pack test --chrome -- --test wasm
+// 3. open developer tools in a browser window
+// 4. navigate to 127.0.0.1:8000
+// 5. verify the request and response by inspection:
+//     - method
+//     - http headers
+//     - outgoing and incoming body
+//
+// #[wasm_bindgen_test]
+// async fn test_browser_fetch() {
```

Comment:
Can't you use a custom cfg or feature so we at least get compile errors?

Reply:
Yes, that'd be much better. In fact all we have to do is comment out the wasm_bindgen_test attribute above instead of the whole function and add an allow dead code and we'll get compile errors. Will do that in my next follow up.